src/main/java/com/danit/services/employee/CompanyService.java

```diff
@@ -18,4 +18,6 @@
 
   boolean companyExists(long id);
 
+  int getCompanyQuant();
```

Comment:
I'd recommend to use either complete words (e.g. 'Quantity') or popular abbreviations ('Qty').